app/views/SMA-prototype/sma.html

```diff
@@ -53,6 +53,12 @@ <h2 class="govuk-heading-l">Recommendation</h2>
          <li>no effective treatments for SMA were identified in an unsymptomatic population, currently only palliative support can be offered - however, the review identified evidence on a new treatment for SMA, called nusinersen, that can improve symptoms in children with SMA</li>
          <li>this evidence review did not find information on the effectiveness of nusinersen in children without symptoms, and there is no evidence on the long term effects of this drug</li>
         </ul>
+        
+        <h2 class="govuk-heading-l">Review cycle</h2>
+        <p class="govuk-body">
+            Last review completed: 2016 <br>
```

Comment:
It's more common to separate lines with their own `<p>` tags than using `<br>`.

I found out that `<br>` have an [accessibility concern](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#Accessibility_concerns).